MANIFEST.in

```diff
@@ -15,3 +15,4 @@ graft scripts
 graft tests
 global-exclude __pycache__
 global-exclude *.py[co]
+prune .github
```

Comment:
Any `MANIFEST` changes need a separate PR, and explanation/justification. (It's just too hot to pull in otherwise.)